packages/cluster-ui/src/sessions/sessionDetails.tsx

```diff
@@ -55,6 +56,7 @@ interface OwnProps {
   refreshNodesLiveness: () => void;
   cancelSession: (payload: ICancelSessionRequest) => void;
   cancelQuery: (payload: ICancelQueryRequest) => void;
+  uiConfig: UIConfigState["pages"]["sessionDetails"];
```

Comment:
`uiConfig` prop has to be considered as optional. Reusing this component in CRDB might require to set it.

Reply:
done